libs/extensions/angular/image-banner/src/image-banner.component.html

```diff
@@ -0,0 +1,84 @@
+<kirby-card *ngIf="externalLink" [themeColor]="backgroundBlur === 'none' ? 'white' : 'dark'">
+  <ng-container *ngTemplateOutlet="sharedCardContent"></ng-container>
+</kirby-card>
+
+<kirby-card
+  *ngIf="!externalLink"
+  [themeColor]="backgroundBlur === 'none' ? 'white' : 'dark'"
+  (click)="bannerClicked($event)"
+>
+  <ng-container *ngTemplateOutlet="sharedCardContent"></ng-container>
+</kirby-card>
+
+<ng-template #sharedCardContent>
+  <div class="blur-image-wrapper">
+    <img class="blur-image" [src]="imagePath" alt="" />
+  </div>
+
+  <!-- When an external link is supplied, this anchor tag expands and fills the entire banner so users can click anywhere or focus the banner -->
+  <a *ngIf="externalLink" class="main-content-anchor" href="{{ externalLink }}" target="_blank"></a>
```

Comment:
Consider using same attribute binding syntax as rest of the file:
```suggestion
  <a *ngIf="externalLink" class="main-content-anchor" [href]="externalLink" target="_blank"></a>
```